### PR TITLE
Initial power controller implementation

### LINKED ISSUE #23: Missing initial implementation of power controller
As an INS node, I would like my power controller to turn off the Wifi module when it is not needed and stay in deep sleep as much as possible so to preserve precious battery life.

DoD:
* Wifi module gets turned on every ~1 minute.
* MCU stays in deep sleep as much as possible.
* State machine with (at least) the following states:
  * `SLEEP_WIFI_OFF` (the wifi module is off, while power controller is in deep sleep and can be woken up by a watchdog timer)
  * `SLEEP_WIFI_ON` (the wifi module is on, while power controller is in deep sleep and can be woken up by a watchdog timer or an external interrupt).
  * `PREPARE_SHUTDOWN` (the power controller warns the Wifi module it is time to shutdown and then it goes to deep sleep for a limited period. The Wifi module can ask to postpone shutdown if it is not ready to)

### PR DESCRIPTION
**Pull request by platisd**
* Initial firmware to run on the ATTIny that controls the power to the WiFi Module
* Test cases to cover the newly introduced functionality

Fixes  #23 

TODO:
* Decide how long to have the power on and how much to sleep. Currently it is set to 16 seconds.

### CODE REVIEW

File: ins-node/hardware/README.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ INS-node board schematics and layout designed with Eagle CAD.
 
 ## List of materials
 * Wemos D1 Mini
-* ATtiny25
+* ATtiny85
 * 2 x AAA battery holders
```
Comment (samup4web) [inline ins-node/hardware/README.md]: I thought we are planning for 3 x AAA batteries now. or?

File: ins-node/README.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ The INS-node is the device that will allow us to pinpoint the location of our co
 ## Setup your IDE
 Follow the instructions found [here](https://wiki.wemos.cc/tutorials:get_started:get_started_in_arduino#using_git_version) to setup your Arduino IDE and enable it to compile and upload sketches for Wemos Mini D1.
 
+To be able to compile for the ATtiny, please follow the instructions [here](http://highlowtech.org/?p=1695).
```
Comment (samup4web) [inline ins-node/README.md:7]: +1 for the links

**Review by samup4web — DISMISSED** [review]
Looks good to me. Good work :+1:

File: ins-node/hardware/README.md (hunk 1 above)
Comment (platisd) [reply ins-node/hardware/README.md]: Indeed we do! 👍 Will fix.

**Comment by platisd** [comment]
Updated the battery holders number. Will require a new review. :smile_cat:

**Comment by platisd** [comment]
@samup4web @lexious89 Reviews please! :smiley:

**Review by lexious89 — APPROVED** [review]
looks good to us.

**Pull request merged**